docs/GettingStarted.md

```diff
@@ -1152,6 +1154,28 @@ Where `options` is an optional `Hash` that accepts the following parameters:
 | `analytics_enabled` | Enable analytics for spans produced by this integration. `true` for on, `nil` to defer to global setting, `false` for off. | `false` |
 | `service_name` | Service name used for `presto` instrumentation | `'presto'` |
 
+### Qless
+
+The Qless integration uses lifecycle hooks to trace job executions.
+
+To add tracing to a Qless job:
+
+```ruby
+require 'ddtrace'
+
+Datadog.configure do |c|
+  c.use :qless, options
+end
+```
+
+Where `options` is an optional `Hash` that accepts the following parameters:
+
+| Key | Description | Default |
+| --- | ----------- | ------- |
+| `analytics_enabled` | Enable analytics for spans produced by this integration. `true` for on, `nil` to defer to the global setting, `false` for off. | `false` |
+| `service_name` | Service name used for `qless` instrumentation | `'qless'` |
+| `workers` | An array including all worker classes you want to trace (e.g. `[MyJob]`) | All jobs |
```

Comment:
> The option you added here, to selectively enable the tracer

@marcotc which option are you referring to?  `analytics_enabled` or `workers`?

Looking at all the other integrations, `analytics_enabled` mostly defaults to false.
By default all worker classes are instrumented, but I'm giving the option to restrict the list (like the resque integration does).

We would not use selective instrumentation.  Would you prefer I remove the `workers` option?

Reply:
Sorry if I wasn't clear. You are correct that I was only referring to `workers`. ` analytics_enabled` is correct.

> We would not use selective instrumentation.

In this case I do suggest the removal of the `workers` option. We try our best to write instrumentation that is safe to enable in the whole application, so restricting instrumentation would only happen in case there there's a strong user case for it.

Reply:
done